### PR TITLE
fix(Forms.Card): remove outline when variant="basic" on Section containers when used in Wizard

### PR DESCRIPTION
**Pull request by langz**
Fixes https://github.com/dnbexperience/eufemia/issues/4308

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Wizard/style/themes/dnb-wizard-layout-theme-ui.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .dnb-forms-wizard-layout {
   &__contents {
     .dnb-card {
+      // something with this adds outline
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Wizard/style/themes/dnb-wizard-layout-theme-ui.scss:4]: Or at least, when removing this, there's no visible outline.
But that of course is not the correct fix, as then many other screenshot tests fails.

Comment (tujoworker) [reply packages/dnb-eufemia/src/extensions/forms/Wizard/style/themes/dnb-wizard-layout-theme-ui.scss:4]: I'll look into it now.

Comment (tujoworker) [reply packages/dnb-eufemia/src/extensions/forms/Wizard/style/themes/dnb-wizard-layout-theme-ui.scss]: Added a CSS specificity fix ✅

**Review by tujoworker — APPROVED** [review]

**Comment by langz** [comment]
@andlbrei, could you please assist me in verifying if this fix solves the issue https://github.com/dnbexperience/eufemia/issues/4308 you reported?

[Here's a CSB reprod using the commit 883a5a28 from this branch/PR](https://codesandbox.io/p/sandbox/nested-cards-outline-broken-in-wizard-forked-39g3pn?workspaceId=9eb653cb-4406-4b8f-bc96-714927cc7fcd)

**Comment by andlbrei** [comment]
@langz Looks like the issue is fixed, indeed 👍

**Pull request merged**